test/Hydration-test.js

```diff
@@ -200,12 +200,33 @@ describe('HydrationLog', function() {
   });
 
   it.skip('can calculate how many fluid ounces a user consumed on each day for the last 7 days', function() {
-    expect(userOneHydrationLog.calculateWeeklyConsumption('2019/06/21')).to.deep.equal([91, 31, 20, 100, 96, 69, 37]);
-    expect(userOneHydrationLog.calculateWeeklyConsumption('2019/06/22')).to.deep.equal([55, 91, 31, 20, 100, 96, 69]);
+    expect(userOneHydrationLog.calculateWeeklyConsumption('2019/06/21')).to.deep.equal(
+      {
+        '2019/06/15': 37,
```

Comment:
Changed this because I think it will be easier to display this weekly data if we have it as object. This way we can maintain easy access to corresponding dates.

Based on what I saw of Chart.js, the logic of displaying data would be something like:
`labels = Object.keys(calculateWeeklyConsumption('2019/06/21'))`
`data = Object.values(calculateWeeklyConsumption('2019/06/21'))`

Although of course there might be intermediate places were we store the object (e.g. in compilation made by `User` method)